usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/whiteboardZoneClient.js

```diff
@@ -0,0 +1,93 @@
+//
+//  drawSphereRemoverClient.js
+//
+//  created by Rebecca Stankus on 03/28/19
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function() {
+    var _this;
+
+    var RGB_MAX_VALUE = 255;
+    var DECIMAL_PLACES = 2;
+
+    var injector;
+    var whiteboard;
+
+    var WhiteboardZone = function() {
+        _this = this;
+    };
+
+    WhiteboardZone.prototype = {
+
+        /* ON PRELOAD: Save a reference to this */
+        preload: function(entityID) {
+            _this.entityID = entityID;
+            whiteboard = Entities.getEntityProperties(_this.entityID, 'parentID').parentID;
```

Comment:
what happens if the user starts off inside the whiteboard zone when the script loads?

Reply:
We do not have a way to handle all cases of what happens if a user is in a zone when a script loads, user crashes while in a zone, changes domains while in a zone, etc. (platform issue). In this case they simply will not have the paint sphere automatically load and will have to manually click a color to begin using the board.

Reply:
hmm, i'm not talking about handling _all_ of the cases, only handling the case of when a user is standing in a zone when the script loads.

i'm ok with inaction on this one, though; the solution of requiring a user to click a sphere manually seems fine to me.